_data/speakers.yml

```diff
@@ -184,4 +184,173 @@ jaidev_deshpande:
\ No newline at end of file
+    FOSS events.</p>
+
+abhishek_mishra:
+  keynote: false
+  name: Abhishek Mishra
+  social:
+    linkedin: https://linkedin.com/in/stalwartcoder
+    twitter: https://twitter.com/stalwartcoder
+  description:
+    <p>
+      I’m Abhishek 👨‍💻, Community first person.
+      I organize community meetups and events.
+      I am also Pythonista and had already given a talk in PyCon India 2019 and PyCon Indonesia 2019.
+      I love to code in python and always give preference to Python when it comes to solve any problem,
+      let it be a web services, desktop application or automation tasks.
+      I am intermediate user in cloud computing and exploring same with python.
+      I’ve been to many community events and conferences, boot-camp for talks and session (Of course related to Python & Dev’s 😎).
+      I try to help Dev communities fullest to my capacity.
+      For more information you can have a look at my LinkedIn profile.
+    </p>
+
+sanket_saurav:
+  keynote: false
+  name: Sanket Saurav
+  social:
+    twitter: https://twitter.com/sanketsaurav
+  description:
+    <p>
+      Sanket (@sanketsaurav) is co-founder at DeepSource.
+      He’s 50% developer and 50% designer.
+      He’s been dabbling with computers since the age of 10, and had started his first venture at 18.
+      He loves the Web and likes building cool stuff that matter.
+      His languages of choice are Python, Go and JavaScript, and he’s been building production apps using these for the past two years.
+      He’s also spoken at more than 50 events and hackathons across the country on open source technologies including Python, HTML5 and web applications in general.
+
+      Sanket also contributes extensively to open-source, with contributions to projects like
+      Django, Celery and Docker, and original Python modules like S3Tree and mimelib.
+    </p>
+
+vijay_yellepeddi:
+  keynote: false
+  name: Vijay Yellepeddi
+  social:
+    github: https://www.github.com/vijaysharmay
+    twitter: https://twitter.com/vijayyellepeddi
+  description:
+    <p>
+      Been writing code from 9+ years.
+      Passionate about Math, Distributed Computing, Graphics Programming & Programming in general.
+      Love exploring new languages and frameworks.
+    </p>
+
+venkata_pingali:
+  keynote: false
+  name: Dr. Venkata Pingali
+  social:
+    linkedin: https://linkedin.com/in/pingali
+    twitter: https://twitter.com/pingali
+  description:
+    <p>
+      Dr. Venkata Pingali is an academic-turned entrepreneur.
+      He works at the intersection of data, ML algorithms, and systems.
+      He builds data products to speed up adoption of ML in the enterprise.
+      Scribble Data, Dr. Pingali’s firm, offers a ML engineering platform and managed service.
+      Dr. Pingali has a BTech from IIT Mumbai and PhD from USC.
+    </p>
+
+pradhvan_bisht:
+  keynote: false
+  name: Pradhvan Bisht
+  social:
+    blog: https://blogs.dgplug.org/pradhvan/
```

Comment:
@ananyo2012 , do we have an svg for blogs that I could use here, worst case I'll go with Medium :P

Reply:
you can try this icon: https://fontawesome.com/icons/blog or the rss icon.

Reply:
Yeah please go ahead with RSS icon. If you don't see it working just use medium for now

Reply:
used medium for now